native/Avalonia.Native/inc/IGetNative.h

```diff
@@ -0,0 +1,5 @@
+class IGetNative
+{
+public:
+    virtual void* GetNative() = 0;
```

Comment:
You can use ObjC++ types in C++ interfaces as long as you include the header from `mm` file.